Formula/opusfile.rb

```diff
@@ -39,7 +39,7 @@ def install
   end
 
   test do
-    testpath.install resource("music_48kbps.opus")
+    resource("sample").stage { testpath.install "gs-16b-1c-44100hz.opus" => "sample.opus" }
```

Comment:
```suggestion
    resource("sample").stage { testpath.install Pathname.pwd.children(false).first => "sample.opus" }
```
Might make it easier to just change `sample` to any other `.opus` file in the future.